File: hw/hidalgo/u1/hw07ChickenFarmSimulatorWi/Egg.js
// ============================================
// Egg Class
// Author: Mikael Hidalgo
// Owner: Mikael Hidalgo
// ============================================

class Egg {
  constructor(size) {
    this.size = size;
  }

  toString() {
    const sizes = { S: "small", M: "medium", L: "large" };
    return `${sizes[this.size] || "unknown"} egg`;
  }
}

module.exports = Egg;

File: hw/hidalgo/u1/hw14SafeInJsonChickenFarmSimulator/ChickenFarmJava/Egg.java
package model;

/**
 *
 * @author Mikael Hidalgo, Paradigm, @ESPE
 */
public class Egg {
    private char size;

    @Override
    public String toString(){
        return "Egg{" + "size=" + size + '}';
    }
    
    public Egg(char size){
        this.size = size;
    }
    
    public char getSize(){
        return size;
    }
    
    public void setSize(char size){
        this.size = size;
    }
}
